ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,13 @@
+Thanks for the feedback! If your issue is related to the Flutter framework itself,
+please open an issue at
+[github.com/flutter/flutter](https://github.com/flutter/flutter/issues/new).
+
+## Steps to Reproduce
+
+_Please tell us what you were doing and what went wrong_
+
+
+## Version info
+
+Please paste the output of running `flutter doctor` here. It will provide the version
```

Comment:
Should we point them to the menu action here (**Tools/Flutter/Flutter Doctor**)?